ictv/templates/storage.html

```diff
@@ -87,65 +93,67 @@ <h1>
                     </th>
                 </tr>
                 <tr>
-                    $code:
-                        total_assets_count = 0
-                        total_assets_size = 0
-                        total_cache_size = 0
-                        for channel in channels:
-                            total_assets_count += channel['assets_count']
-                            total_assets_size += channel['assets_size']
-                            total_cache_size += int(channel['cache_percentage']*channel['assets_size'])
-                        total_asset_pc = ((total_assets_size - total_cache_size)*100/total_assets_size) if total_assets_size else 0
-                        total_cache_pc = (total_cache_size*100/total_assets_size) if total_assets_size else 0
+                        {% set total_assets_count = 0 %}
+                        {% set total_assets_size = 0 %}
+                        {% set total_cache_size = 0 %}
+                        {% for channel in channels %}
```

Comment:
Did not check at runtime but beware, jinja has some scope subtleties that prevents some changes in loops without using  namespaces.

Reply:
This needs to be checked indeed.